teams.json

```diff
@@ -14,11 +14,13 @@
     {
       "name": "code-build-run",
       "apis": [
-        "ndb",
         "appengine",
+        "cloudfunctions",
+        "cloudtasks",
+        "compute",
```

Comment:
While CBR does technically cover that area, this is a dirty space right now.  I think we should drop compute from here and discuss offline

Reply:
Done in a927513